geometry_msgs/PoseStamped location

---

Scene scene
